app/javascript/src/components/Dashboard/Contacts/Table/ColumnRowDefinitions.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
This is not a file that exports any React components and thus it doesn't make sense to name this file in PascalCase.

Except the `COLUMN_DATA` all other things ought to be moved to `constants.js`.

Refer: https://github.com/eshaanks/neeto-ui-challenge/pull/8#discussion_r971923751

Reply:
After shifting all except COLUMN_DATA how should we name the file?

Reply:
@eshaanks After doing that this file only contains two literal constants. Thus this file should be named `constants.js`.

Reply:
Yedhin , I might be misunderstanding  this. Since the reason why I shifted the code from constants.js file in the first place to avoid having 'jsx' code in a 'js' file. So after moving the literal constants to constants.js (the ROW_DATA and DUMMY_CONTACT_DATA) we are left with COLUMN_DATA but it has 'jsx'. So should I name it constants.jsx or something else. 
PS: I have changed the column definitions to show delete alerts on delete click, hence now it is a function. Therefore i have named it ColumnDefinition as we would name a react component file?

Reply:
Moved ROW_DATA and DUMMY_CONTACT_DATA constants to constants.js, Moved the getColumnDefinition function to utils.jsx (renamed utils.js to utils.jsx) and rectified all pertinent imports.